lib/link_checker/uri_checker/http_checker.rb

```diff
@@ -180,7 +178,7 @@ def check_meta_mature_rating
 
     def check_google_safebrowsing
       api_key = Rails.application.secrets.google_api_key
-      return unless api_key
+      return if !Rails.env.production? || !api_key
```

Comment:
I think we should move this check above to where we call it and then raise an exception here if the `api_key` is not set in the method.

```ruby
check_google_safebrowsing unless Rails.env.production? || Rails.application.secrets.google_api_key
```

```ruby
raise "Missing API key" unless api_key
```